packages/p2p/src/components/advertiser-page/advertiser-page.jsx

```diff
@@ -3,13 +3,14 @@ import { Loading, Text } from '@deriv/components';
 import { daysSince, isMobile } from '@deriv/shared';
 import { reaction } from 'mobx';
 import { observer } from 'mobx-react-lite';
-import PageReturn from 'Components/page-return/page-return.jsx';
+import { useStores } from 'Stores';
 import { Localize, localize } from 'Components/i18next';
 import { buy_sell } from 'Constants/buy-sell';
 import RateChangeModal from 'Components/buy-sell/rate-change-modal.jsx';
 import BuySellModal from 'Components/buy-sell/buy-sell-modal.jsx';
+import PageReturn from 'Components/page-return/page-return.jsx';
+// import RecommendedBy from 'Components/recommended-by';
```

Comment:
can we remove this?

Reply:
My last commit wasn't pushed after I fixed the conflicts 😅